plugins/modules/silence.py

```diff
@@ -32,6 +32,8 @@
   - sensu.sensu_go.state
```

Comment:
Can you please add a note documenting that at least one of (`subscription`, `check`) is required?